services/collaboration/pkg/middleware/wopicontext.go

```diff
@@ -36,6 +39,21 @@ type WopiContext struct {
 	FileReference     *providerv1beta1.Reference
 	TemplateReference *providerv1beta1.Reference
 	ViewMode          appproviderv1beta1.ViewMode
+	scope             map[string]*auth.Scope
+}
+
+// GetScope returns the scope from the WopiContext
```

Comment:
We might need some additional information to say that you need to provide a "message" and the result will be written there.

In general, parameters are treated as read-only, so nobody expects a parameter to change inside the function. The cases where this happens are limited. If the idea is that you just need to pass a placeholder where the result will be written, I think it's better to document it to avoid confusion.